content/_partials/changelog-changes.html.haml

```diff
@@ -23,7 +23,7 @@
               - if index != 0
                 ,&nbsp;
               - if reference.issue
-                %a{:href => "https://issues.jenkins.io/browse/JENKINS-#{reference.issue}" }<>
+                %a{:href => "https://github.com/jenkinsci/jenkins/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-#{reference.issue}%22" }<>
```

Comment:
e.g https://github.com/timja-org/jenkins-gh-issues-poc-11-14/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-75990%22